### ISSUE TITLE
Participação na hacktoberfest

### ISSUE THREAD

**Issue by rafaelbernard** [php/doc-pt_br#183]
Oi, pessoal.

O que acham de marcar o repositório para participação na Hacktoberfest? Pode chamar mais contribuidores.

Maintainers: https://hacktoberfest.com/participation/#maintainers

Abraços,
Rafael

**Comment by rafaelbernard** [comment]
Concordo inteiramente. Fiquei enrolado e acabei demorando a colocar a mensagem aqui.

Basta adicionar o label `hacktoberfest` no repositório. Acho que vale já a pena, pois poderíamos, no mínimo, anunciar nas redes que o repositório possui a label. Ainda podem ter contribuidores nesta edição. E não há nada a perder.

**Comment by alfsb** [comment]
Bom, pelo lado bom tivemos boas experiências com `hacktoberfest`. Pelo lado ruim, está bem em cima, com uma festa de encerramento já amanhã: <https://try.digitalocean.com/hacktoberfest-wrap-party/>. O outro ponto é que não temos acesso para criar o tópico no repositório, teria de pedir ao PHP Group para isso.

Mas sim, bora encaminhar isso. Pode um de vocês abrir um issue no doc-en, pedindo o a inclusão e a permanência do tópico no repositório aqui.

Só deem um toque nesse issue antes, para evitar abrir duplicado (mesmo sistema de reserva dos wips).

E daí por na agenda, desde já, para o ano que veem fazermos uma organização melhor.

**Comment by rafaelbernard** [comment]
Outra opção é adicional a label nos PRs que foram abertos em outubro (mesmo que sejam mesclados depois). Sei que está em cima, mas pode gerar um bom sentimento para pessoas que participam do hacktoberfest ver que PRs abertos simplesmente apareceram como participantes na festa.

Se as labels forem incluídas agora, todos os PRs serão considerados retroativamente.

**Comment by rafaelbernard** [comment]
> Mas sim, bora encaminhar isso. Pode um de vocês abrir um issue no doc-en, pedindo o a inclusão e a permanência do tópico no repositório aqui.

Beleza. Vou fazer isso.

**Comment by rafaelbernard** [comment]
Feito! :-D

**Comment by rafaelbernard** [comment]
Vou fechar este caso e, claro, colocar na agenda para o ano que vem, pois pode ser muito útil para nós. Vi o repositório `oh-my-zsh`, onde já contribuí, e o número de PRs abertos para outubro é notadamente maior, possivelmente em razão do tópico.
